.github/workflows/test.yml

```diff
@@ -261,6 +262,9 @@ jobs:
       - name: Python3
         run: sudo lxc exec test-container --env DEBIAN_FRONTEND=noninteractive -- apt-get -y install -y python3
 
+      # Install LXD inside cotnainer its missing inside container
```

Comment:
s/cotnainer/container/

Reply:
It is a comment. I would leave it and it can be fixed in another commit if the rest of the patch is good.